exercises/difference-of-squares/example.py

```diff
@@ -1,11 +1,12 @@
-def square_of_sum(count):
-    sum_ = count * (count + 1) / 2
+def square_of_sum(number):
+    sum_ = number * (number + 1) / 2
     return sum_ * sum_
 
 
-def sum_of_squares(count):
-    return sum(m * m for m in range(count + 1))
+def sum_of_squares(number):
+    numerator = number * (number + 1) * (2 * number + 1)
+    return numerator/6
```

Comment:
Your `flake8` setup didn't complain about this? That is strange.

```suggestion
    return numerator / 6
```

Reply:
I just ran `Flake8` again on the branch....**it did not complain!**.   Neither did PyCharm.  Is there a config file you can share for `Flake8`?  Currently I am using all defaults and setting `max-complextiy` to 10.  PyCharms `PEP8` enforcement appears to be turned on, but it only complains if the space around an operator is **asymmetric** - it doesn't care if the space is _missing_, only that it _matches_.  Grr.  _In any case_  noted for next time!